server/form-pages/apply/risk-and-need-factors/further-considerations/contingencyPlanQuestions.test.ts

```diff
@@ -1,12 +1,20 @@
 import { ContingencyPlanQuestionsBody } from '../../../../@types/ui'
+import { addResponseToApplication } from '../../../../testutils/addResponseToApplication'
 import applicationFactory from '../../../../testutils/factories/application'
+import contingencyPlanPartner from '../../../../testutils/factories/contingencyPlanPartner'
 import contingencyPlanQuestionsBodyFactory from '../../../../testutils/factories/contingencyPlanQuestionsBody'
 import { itShouldHaveNextValue, itShouldHavePreviousValue } from '../../../shared-examples'
 import ContingencyPlanQuestions from './contingencyPlanQuestions'
 
 describe('ContingencyPlanQuestions', () => {
   const body = contingencyPlanQuestionsBodyFactory.build()
-  const application = applicationFactory.build()
+  const contingencyPlanPartners = contingencyPlanPartner.buildList(2)
+  const application = addResponseToApplication(applicationFactory.build(), {
```

Comment:
What's the difference between `addResponseToApplication` and `withPageResponse`? I think using `withPageResponse` is clearer - perhaps even with an additional `withContingencyPlanPartners` helper?

Reply:
`addResponseToApplication` *adds* a response to the application preserving the rest of the `data` values whereas when generating an application from the factory we can only add the specified responses at the time we create the application. 
I *think* we need `addResponseToApplication` as it's used in `contingencyPlanPages.test.ts` but you're right that we don't need it here and it's tidier to use `withContingencyPlanPartners`